server/routes/payment.py

```
@@ -1,49 +1,417 @@
+        print(f"DEBUG: Invoice object: {invoice}")
+        print(f"DEBUG: Invoice payments attribute: {hasattr(invoice, 'payments')}")
+
+        # ===== DEBUG: LIST ALL PAYMENTS IN DATABASE =====
+        print("=" * 60)
+        print("DEBUG: ALL PAYMENTS IN DATABASE")
+        print("=" * 60)
+        try:
+            all_payments = Payment.query.all()
+            print(f"Total payments in database: {len(all_payments)}")
+
+            for i, payment in enumerate(all_payments):
```

Ruff (0.13.2)

Loop control variable `i` not used within loop body

(B007)